TypeCobol/Common.props

```diff
@@ -42,6 +42,7 @@
     <WarningLevel>4</WarningLevel>
 	<CodeAnalysisRuleSet>TypeCobol.ruleset</CodeAnalysisRuleSet>
     <RunCodeAnalysis>true</RunCodeAnalysis>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Can you put the LangVersion at the top of this file, so it applies for all configuration ?